scripts/zsh.pl

```diff
@@ -1,19 +1,25 @@
 #!/usr/bin/env perl
 
-# Generate ZSH completion
+# Generate ZSH or FISH completion
```

Comment:
I'm not sure it's worth complicating this script with multiple shell support, instead of just make a new `fish.pl` file. At the very least we should rename it..

Reply:
Thanks for your remarks! I also considered a separate `fish.pl`, but refrained from doing so to avoid duplicating the code for parsing and escaping the `curl --help` output. Ad renaming: agreed!